.circleci/config.yml

```diff
@@ -198,73 +150,20 @@ jobs:
           paths:
             - ~/.ccache
 
-  build-release-binary-osx:
```

Comment:
same, these were only run on git tags and no one cares about their success or failure